### ISSUE TITLE
Learning rate scheduler is not used in train.py

### ISSUE THREAD

**Issue by ehsan-ami** [georgeliu233/STrajNet#3]
Hi,

Thanks for releasing the code!

I get into of issues: 

1-The learning rate scheduler is not used in training. According to the paper, LR decay 50% every 3 epoch. However, this kind of scheduler is not used in the released code. 
2-The paper states that the Adam optimizer is used, while in the code, Nadam optimizer is used. 

Could you please release the version corresponding code to the paper to make it reproducible as a baseline?

**Comment by georgeliu233** [comment]
Hi, Thanks for your comments. Regarding the issues above, it is a mistake that we used in the original Waymo technical report. More details for the change:1) We used the cosine-restart scheduler instead of step-lr decay, which shows limited improvements compared to the original lr schedule.  2) it should be NAdam instead of Adam.

**Comment by ehsan-ami** [comment]
Great! Thanks so much for the clarification!
